### PR TITLE
Add Ubuntu 20.04 build dependencies list

### PR DESCRIPTION
**Pull request by martyngigg**
**Description of work.**

Some users have asked how to build on Ubuntu 20.04 as we don't support it yet. This adds the list of required packages to the developer docs to save reproducing the same list in emails.

**To test:**

Docs Review.

*There is no associated issue.*

*This does not require release notes* because **it is a minor documentation update.**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: dev-docs/source/GettingStarted.rst [hunk 1]
```diff
@@ -90,6 +90,61 @@ where ``X.Y.Z`` should be replaced with the version that was downloaded.
 
 if you wish to setup eclipse for use developing mantid, then instructions can be found :ref:`here <Eclipse>`.
 
+Ubuntu 20.04
+~~~~~~~~~~~~
+- Mantid is not yet officially supported on Ubuntu 20.04 as Qt4 has been removed but Workench can be built by installing:
```
Comment (DanielMurphy22) [inline dev-docs/source/GettingStarted.rst:95]:
```suggestion
- Mantid is not yet officially supported on Ubuntu 20.04 as Qt4 has been removed but Workbench can be built by installing:
```

Comment (DanielMurphy22) [reply dev-docs/source/GettingStarted.rst:95]: Sorry, great to have these instructions, but I think it's worth correcting this typo.

**Review by DanielMurphy22 — CHANGES_REQUESTED** [review]

File: dev-docs/source/GettingStarted.rst (hunk 1 above)
Comment (martyngigg) [reply dev-docs/source/GettingStarted.rst:95]: Oh yes absolutely! Thanks!

**Review by DanielMurphy22 — APPROVED** [review]
Thanks!

**Pull request merged**